.github/workflows/links_checker.yml

```diff
@@ -16,22 +16,16 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Checkout OpenSearch-Dashboards
-      uses: actions/checkout@v2
-      with:
-        repository: opensearch-project/OpenSearch-Dashboards
-        ref: main
-        path: OpenSearch-Dashboards
     - name: Load Excludes
       run: |
-        LYCHEE_EXCLUDE=$(sed -e :a  -e 'N;s/\n/ --exclude /;ta' OpenSearch-Dashboards/.lycheeexclude)
+        LYCHEE_EXCLUDE=$(curl -fsSL https://raw.githubusercontent.com/opensearch-project/OpenSearch-Dashboards/HEAD/.lycheeexclude | cat - .lycheeexclude | grep -v '^#\|^$' | sed -e :a  -e 'N;s/\n/ --exclude /;ta')
```

Comment:
Shall we have a repo-specific `.lycheeexclude` instead of rely on the external? I guess we can start with a small list of `.lycheeexclude` first?

Reply:
it's already in the PR
https://github.com/opensearch-project/dashboards-assistant/blob/baa1ab9539e7605a24722ce35071c97a83657a4d/.lycheeexclude#L1-L5

Reply:
Yes, but do we still need to download from `https://raw.githubusercontent.com/opensearch-project/OpenSearch-Dashboards/HEAD/.lycheeexclude`?

Reply:
we don't want to maintain common excludes in two places, the mainly part should from OSD core and we have patch in this repo.

Reply:
> > Yes, but do we still need to download from `https://raw.githubusercontent.com/opensearch-project/OpenSearch-Dashboards/HEAD/.lycheeexclude`?
> 
> we don't want to maintain common excludes in two places, the mainly part should from OSD core and we have patch in this repo.

But the `.lycheeexclude` in OSD repo is not common excludes as I understand, it's specific to OSD repo, am I right?

Reply:
> it's specific to OSD repo

agree, it is specific to OSD. But dashboards-assistant is a plugin of OSD, `.lycheeexclude` defined in OSD should suitable for plugins.

Reply:
i think the OSD list makes sense, but we are the only one using it besides OSD. It was added by https://github.com/opensearch-project/dashboards-assistant/commit/db292f8e115f327db14d1c0d6e4431a30fe0b819, my PR just avoids the whole repo clone for faster CI and debugging. @SuZhou-Joe any specific reason that you added the `.lycheeexclude` from OSD?

Reply:
Actually I copy the lint-checker from OSD, and thus I use the .lycheeexclude from OSD as a reference to keep dashboards-assistant align with OSD. It is OK that we remove this step or merge some repo-specific exclude config with OSD's.

Reply:
i see, in that case i'll remove it. the OSD one contains links like `github.com`, it excludes 404 checks for github links in this repo. without it CI would be able to catch outdated github URLs